chore(docs): Deployment Updates & Cleanup (#37038)
gatsbyjs/gatsby: docs/docs/how-to/previews-deploys-hosting/deploying-to-iis.md

title: Deploying to Microsoft Internet Information Server (IIS)
---

[Internet Information Services (IIS)](https://www.iis.net/) for Windows® Server is a flexible, secure and manageable Web server for hosting anything on the Web. From media streaming to web applications, IIS's scalable and open architecture is ready to handle the most demanding tasks.

## Prerequisites

- A Gatsby project set up. (Need help creating one? Follow the [Quick Start](/docs/quick-start/))

## Instructions

### wwwroot

> Deploying to IIS is mostly copy & paste. After `gatsby build`, copy the contents of the `public/` folder into the `wwwroot` folder of your IIS web.

Reviewer: ```suggestion
Deploying to IIS is a matter of running `gatsby build` and copying the contents of your local `public` folder to the `wwwroot` folder of your IIS web server.
```

I know this wasn't changed in this PR, but figured it's enough in scope to suggest
